extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampExtractorHelper.java

```diff
@@ -54,7 +54,8 @@ public static String getStreamUrlFromIds(final long bandId,
 
     /**
      * Fetch artist details from mobile endpoint.
-     * <a href=https://notabug.org/fynngodau/bandcampDirect/wiki/rewindBandcamp+%E2%80%93+Fetching+artist+details>
+     * <a href="https://notabug.org/fynngodau/bandcampDirect/wiki/
+     * rewindBandcamp+%E2%80%93+Fetching+artist+details">
```

Comment:
Does it really still work?!?

Reply:
In IDEA, yes, because of the `"` characters I added. In other IDEs, I don't know.